docs/api/users.md

```diff
@@ -46,7 +53,7 @@ Content-Type: application/json
   "firstName": "Najat",
   "infix": "van der",
   "lastName": "Lee",
-  "tenantId": null,
+  "organisationId": null,
```

Comment:
Couldn't we assume that any User that is retrieved / created is part of the current Organisation, so an organisationId is technically redundant?